lib/utils.js

```diff
@@ -199,9 +199,17 @@ function prettifyLevel ({ log, colorizer = defaultColorizer }) {
 function prettifyMessage ({ log, messageFormat, messageKey = MESSAGE_KEY, colorizer = defaultColorizer }) {
   if (messageFormat) {
     const message = String(messageFormat).replace(/{([^{}]+)}/g, function (match, p1) {
-      if (p1 && log[p1]) {
+      if (p1 && log[p1] && typeof log[p1] !== 'object') {
```

Comment:
Please use `isObject` https://github.com/pinojs/pino-pretty/blob/852fdf707c1a57627a3f523b0cec8824e5d24ab4/lib/utils.js#L78-L80

Reply:
Had fixed it. The issue is #124